### DISCUSSION TITLE
Controlling menu in a Navbar

### DISCUSSION THREAD

**Discussion by jpablo** [saadeghi/daisyui#1478] [Q&A]
Hi, I'm trying to find out how to programmatically control a menu in a navbar. Specifically:

1. How to prevent the dropdown from appearing at all. I'm using a class `disabled` on the container `<li>` which disables the menu but it still appears on hover.

2. After clicking on a menu entry, I'd like to close the menu programmatically.

Thanks!

**Comment by saadeghi** [comment]
Can you please give me a sample code?
